build_tools/cmake/run_tf_tests.sh

```diff
@@ -14,6 +14,7 @@ cd "${ROOT_DIR}"
 BUILD_DIR="$1"
 IREE_VULKAN_DISABLE="${IREE_VULKAN_DISABLE:-0}"
 IREE_LLVM_CPU_DISABLE="${IREE_LLVM_CPU_DISABLE:-0}"
+IREE_VMVX_TESTS_ENABLE="${IREE_VMVX_TESTS_ENABLE:-0}"
```

Comment:
Why default "enable" to off? How about matching the other `_DISABLE` options? Then no need to `--env IREE_VMVX_TESTS_ENABLE=1` in ci.yml.

Reply:
vmvx tests take longer to run (not as optimized as llvmcpu/gpu). The added 4 tests  in this PR are okay-ish, but if we add more examples later it could add more overhead to the overall CI time.

Reply:
Ah, I see that what's left in https://github.com/openxla/iree/tree/main/integrations/tensorflow/test/iree_tfl_tests is only large, full program tests... 

Can you add a comment explaining why VMVX is disabled by default? That's normally the opposite of what we'd want from a portable/reference backend.

Reply:
Done.

Reply:
Even if it's disabled by default, I think it's confusing to have the option be different from the others. We have other disable options that are true by default. I actually think it's a bit confusing that they're all negative booleans, but if we're going to switch them, let's switch them all.

Vulkan FP16 and CUDA are disabled by default here, for instance: https://github.com/openxla/iree/blob/a50bc65c6c3e75a1122d64f1575fd2791be64e8e/build_tools/cmake/ctest_all.sh#L35

I'd also drop the "TESTS" part of the name and have it match all the others. So

```suggestion
IREE_VMVX_DISABLE="${IREE_VMVX_DISABLE:-1}"
```

Reply:
Thanks. Use disable logic instead.